modules/raft/src/integrationTest/java/org/apache/ignite/raft/jraft/core/ItNodeTest.java

```diff
@@ -3506,6 +3507,102 @@ public void onRawConfigurationCommitted(ConfigurationEntry conf) {
         );
     }
 
+    @Test
+    public void testIndexAndTermOfCfgArePropagatedToSnapshotMeta() throws Exception {
```

Comment:
The scenario is pretty lengthy and complex. Could you please elaborate on it in the javadoc? What it tests for, what the scenario is, what verifications are made...

Reply:
added